cmd/gopm/testdata/config-dump-tags.txt

```diff
@@ -0,0 +1,65 @@
+mkdir progdir
+gopm -c gopmconfig --quit-delay 0 -t foo=one --inject bar=two &
+waitfile gopm.socket
+gopmctl --addr unix:gopm.socket dump-config
```

Comment:
Rather than using `dump-config` which is a bit of a pain as the expected output needs to be adjusted whenever any detail of the configuration changes, how about just running the command and checking that its output is as expected?

Reply:
The purpose of using `dump-config` was to verify the tags correctly injected their substitute values.